warehouse-management-service/shelf.go

```diff
@@ -0,0 +1,26 @@
+package wms
+
+import (
+	"errors"
+)
+
+type Shelf struct {
```

Comment:
As noted above, this struct can replace `CreateShelfRequest` & `UpdateShelfRequest`